docs/en/operation/Scalability-Tuning.md

```diff
@@ -111,6 +111,8 @@ clients.
 * Amount of network bandwidth required by the worker - We recommend at least 10 MB/s per concurrent
 client. This resource is less important if a majority of tasks have locality and use short circuit.
 
+The metric "Worker.ActiveClients" displays the current number of clients connected to a particular worker,
```

Comment:
I guess it makes sense to mention this is added in version blah. It's not too intuitive for the user to compare the latest doc to the doc for a previous version to note this difference.

Reply:
it was merged 2 days ago.. so it is hard to put a version next to it.. 2.7?

Reply:
What I used to do (for example https://docs.alluxio.io/os/user/stable/en/deploy/Running-Alluxio-On-Kubernetes.html#basic-setup mentioned `2.3`) is referring to the next version, but now the versioning policy has changed. So might need @apc999 to confirm on the next version and some general guidance on below:

For a user (and oftentimes we are users too, to feature we are not familiar enough about), it's natural to go to the latest feature doc, and later downloading a versioned tarball which may not necessarily match. I think it's helpful to associate a version, at least for features that only exist in the latest version number (`edge` or `SNAPSHOT`). What do you think @apc999 ?

Reply:
What I used to do (for example https://docs.alluxio.io/os/user/stable/en/deploy/Running-Alluxio-On-Kubernetes.html#basic-setup mentioned `2.3`) is referring to the next version, but now the versioning policy has changed. So might need @apc999 to confirm on the next version and some general guidance on below:

For a user (and oftentimes we are users too, to feature we are not familiar enough about), it's natural to go to the latest feature doc, and later downloading a versioned tarball which may not necessarily match. I think it's helpful to associate a version, at least for features that only exist in the latest version number (`edge` or `SNAPSHOT`). What do you think @apc999 ?

Reply:
i think i will remove this comment for now, and insert it once we have a release.